### ISSUE TITLE
Undefined array key warnings when editing translated tt_content record

### ISSUE THREAD

**Issue by Physikbuddha** [b13/container#467]
In v12, opening a content element in the backend in a language different than the default language, two PHP warnings are issued:

```
PHP Warning: Undefined array key "tx_container_parent" in /var/www/html/vendor/b13/container/Classes/Tca/ItemProcFunc.php line 51
PHP Warning: Undefined array key "tx_container_parent" in /var/www/html/vendor/b13/container/Classes/Tca/ItemProcFunc.php line 83
```

This is caused by `BackendUtility::getProcessedValue()` calling `BackendUtility::getLabelsFromItemsList()` with a fake `$row` parameter in line 1658, only containing `uid` and `pid`.

The stacktrace is as follows:

![2024-01-22 14_18_24](https://github.com/b13/container/assets/8708554/f215e799-88f6-4b80-a853-14995a6cbdb0)

**Comment by W4ldgeist** [comment]
And the fix would be to add an isset check, before accessing the array key on Line 51:
`if (isset($row['tx_container_parent']) && $row['tx_container_parent'] > 0) {`
and Line 83:
`if (isset($row['tx_container_parent']) && $row['tx_container_parent'] > 0) {`

**Comment by m-kappenberg** [comment]
Maybe we have always to check if an array key exists un PHP8.
My most unwantted PHP change + my most forgotten ;-)

**Comment by DavidBruchmann** [comment]
`if (!empty($key) && ...)`

**Comment by achimfritz** [comment]
should be fixed in 2.3.3 with https://github.com/b13/container/commit/2ef3fd19c9c5a71c25a92420e26e2d46808ea358
